Tests/SwiftSyntaxTest/SyntaxTests.swift

```diff
@@ -25,4 +25,19 @@ public class SyntaxTests: XCTestCase {
     XCTAssertTrue(TokenListSyntax([TokenSyntax.funcKeyword(presence: .missing)]).hasError)
     XCTAssertFalse(TokenListSyntax([TokenSyntax.funcKeyword(presence: .present)]).hasError)
   }
+
+  public func testDetach() {
+    let s = StructDeclSyntax(
+      attributes: nil,
```

Comment:
Not really related to this PR but can’t we omit these `nil` arguments now?

Reply:
No, I haven't merged that PR yet.